controllers/machine_controller.go

```diff
@@ -285,7 +284,8 @@ func (r *MachineReconciler) drainNode(ctx context.Context, cluster *clusterv1.Cl
 	if err != nil {
 		if apierrors.IsNotFound(err) {
 			// If an admin deletes the node directly, we'll end up here.
-			klog.Infof("Machine %v: Could not find node %v from noderef, it may have already been deleted: %v", machineName, nodeName, cluster.Name)
+			klog.Infof("Machine %s/%s: Could not find node %v from noderef, it may have already been deleted: %v",
+				cluster.Namespace, machineName, nodeName, cluster.Name)
```

Comment:
Cluster.Name feels a bit odd here, it looks like this should be `err` instead based on the use in the string above.